colour/plotting/common.py

```diff
@@ -494,7 +494,13 @@ def artist(**kwargs: KwargsArtist | Any) -> Tuple[Figure, Axes]:
 
         return figure, figure.gca()
     else:
-        return plt.gcf(), cast(Axes, axes)
+        axes = cast(Axes, axes)
+        figure = axes.figure
+
+        while isinstance(figure, SubFigure):
```

Comment:
Why are we using `while` here?

Reply:
When I first read the docs I thought the get figure might sometimes return a parent subfigure